mlx/traceability/assets/traceability.js

```diff
@@ -48,14 +48,40 @@ $(document).ready(function () {
                     // lines up the caption behind the item ID
                     caption.css('transform', 'translate(0.3rem, -3px)');
                 }
-            }, function() {
+            }, function () {
                 // leaving hover state
                 caption.css('transform', 'none');  // resets the transformations
                 caption.hide();
             }
         );
     });
 
+    $('table > tbody > tr[class^=item-group-]').each(function (i) {
+        $(this).on("contextmenu",
+            function (event) {
+                event.preventDefault()
+                var groupName = /item-group-\d+/.exec($(this).attr('class'))[0];
+                $(this).parent().find(`tr.${groupName} > td > p.item-link`).each(function (j) {
+                    const cell = $(this).parent();
+                    const maxWidth = Math.max(cell.width(), ($("div.rst-content").width() / 2));
+                    cell.css("maxWidth", maxWidth);
+                    const content = $(this).children('div.content').first();
+                    if (content.length) {
+                        content.toggle();
+                    } else {
+                        var link = $(this).children('a').first();
+                        var container = $('<div>', { class: 'content' });
+                        var paragraph = $(this)
+                        container.load(link.attr('href').replace('#', ' #content-'), function () {
```

Comment:
This `jQuery.load` fetches the entire HTML page every time and extracts the element with the specified id. I wonder if there is a better alternative.

Reply:
This is fine. The Ajax requests are cached by the browser.